diesel/src/expression/mod.rs

```diff
@@ -47,9 +47,12 @@ pub mod sql_literal;
 pub mod subselect;
 
 #[doc(hidden)]
+#[allow(non_camel_case_types)]
 pub mod dsl {
+    use dsl::SqlTypeOf;
+
     #[doc(inline)]
-    pub use super::count::{count, count_star};
+    pub use super::count::*;
```

Comment:
pub using items with `#[doc(inline)]` does not actually show the documentation in that place (Rustdoc will only render `pub use super::count::*;` here not the actual imported types.

Reply:
I don't think that's correct. http://docs.diesel.rs/diesel/dsl/index.html https://github.com/diesel-rs/diesel/blob/master/diesel/src/lib.rs#L187-L191

Reply:
Hmm, then this only works inside of an crate. (There was [this PR](https://github.com/diesel-rs/diesel/pull/1410) for diesel_migrations)

Reply:
Correct, you cannot inline documentation across crates.